OrionUO/Managers/SoundManager.cpp

```diff
@@ -8,7 +8,7 @@
 #endif
 
 #if !USE_BASS
-#define WITH_SDL2
+#define WITH_SDL2_STATIC
```

Comment:
What are you fixing with this? This worked fine.

Reply:
On mac os that dynamic version of ass is trying to load .so files to map functions. That can’t work since mac os sdl is using dylib (i have one from brew), but even when i tried to load proper file it does not help, so i simple used this one. Is there any reasons not to use static?

Reply:
Someone on discord got mac building this way without problem. I'll check this again, I do not think this will be an issue anyway as we do not really link statically with sdl2 and it will be already loaded anyway.